### PR TITLE
implement get services

### PR DESCRIPTION
**Pull request by wangqj**

### CODE REVIEW

File: spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java [hunk 1]
```diff
@@ -270,6 +271,9 @@ public boolean deRegisterInstance(String serviceId, String instanceId)
         objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         LOGGER.info(response.getContent());
         Microservice result = objectMapper.readValue(response.getContent(), Microservice.class);
+        if (result == null || result.getInstances() == null) {
+          return instanceList;
```
Comment (rainbowzhang) [inline spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java]: I think this line should add logs before return statement for exception branch.

File: spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java [hunk 2]
```diff
@@ -365,6 +369,29 @@ public boolean updateInstanceStatus(String serviceId, String instanceId, String
...
+  public MicroserviceResponse getServices() throws ServiceCombException {
+    MicroserviceResponse result = null;
+    Response response = null;
+    try {
+      response = httpTransport.sendGetRequest(
+          buildURI("/registry/microservices"));
+      if (response.getStatusCode() == HttpStatus.SC_OK) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        LOGGER.info(response.getContent());
```
Comment (rainbowzhang) [inline spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java]: response.getContent() may  be needed more time costly, and can add if condition like this :if(isdebug)(debug(info))

Comment (wangqj) [reply spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java]: done

File: spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java (hunk 1 above)
Comment (wangqj) [reply spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java]: done

**Pull request merged**